cmake/FindLibBpf.cmake

```diff
@@ -3,21 +3,17 @@
 #
 #  LIBBPF_FOUND            - system has libbpf
 #  LIBBPF_INCLUDE_DIR      - the libbpf include directory
-#  LIBBPF_SOURCE_DIR       - the libbpf source directory
+#  LIBBPF_STATIC_LIBRARIES - the libbpf source directory
 #  LIBBPF_LIBRARIES        - link these to use libbpf
 
-#if (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_SOURCE_DIR)
+#if (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_STATIC_LIBRARIES)
 #  set (LibBpf_FIND_QUIETLY TRUE)
-#endif (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_SOURCE_DIR)
+#endif (LIBBPF_LIBRARIES AND LIBBPF_INCLUDE_DIR AND LIBBPF_STATIC_LIBRARIES)
 
 # You'll need following packages to be installed (Fedora names):
 # libbpf
 # libbpf-debugsource
```

Comment:
The above `libbpf-debugsource` can be removed.

Reply:
missed that one, sry, fixed